### PR TITLE
[UI] Login and Sign-Up | Common layout file Added

### PR DESCRIPTION
**Pull request by imjalpreet**
@nisargjhaveri @gauravmittal1995 Have a look

### CODE REVIEW

File: app/views/auth/signup.php [hunk 1]
```diff
@@ -1,11 +1,124 @@
+<!DOCTYPE html>
+<html>
 <?php
 if (!empty($error)) {
     echo $error;
```
Comment (nisargjhaveri) [inline app/views/auth/signup.php]: Style this too?

File: app/views/auth/login.php [hunk 2]
```diff
@@ -1,10 +1,116 @@
+<!DOCTYPE html>
+<html>
 <?php
 if (!empty($error)) {
     echo $error;
```
Comment (nisargjhaveri) [inline app/views/auth/login.php]: Style this too?

**Comment by imjalpreet** [comment]
Done!

**Comment by gauravmittal1995** [comment]
what about roles? we need to allow signup if the current person is signed in and his role is admin

**Comment by nisargjhaveri** [comment]
See #94. We can add the same thing on signup also.

**Comment by imjalpreet** [comment]
@nisargjhaveri  @gauravmittal1995 Test it

File: app/views/requests/view.php [hunk 3]
```diff
@@ -16,59 +16,10 @@
...
-                    </a>
-                </li>
-            </ul>
-
-        </div><!-- /.navbar-collapse -->
-    </div><!-- /.container-fluid -->
-</nav>
+<?php
+$path = $_SERVER['DOCUMENT_ROOT'];
+include ($path.'/app/views/navbar.php');
```
Comment (nisargjhaveri) [inline app/views/requests/view.php]: We should use `$this->load_fragment('navbar');` instead of these two lines.

**Comment by nisargjhaveri** [comment]
If possible, create two separate PRs.

File: app/views/auth/login.php [hunk 4]
```diff
@@ -1,10 +1,115 @@
...
+    <link href="<?php echo base_url(); ?>static/css/sidebar.css" rel="stylesheet" media="screen">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.2/jquery.min.js"></script>
+    <script src="<?php echo base_url(); ?>static/js/bootstrap.js"></script>
+
+</head>
+<nav class="navbar navbar-inverse navbar-fixed-top" id="navbar" role="navigation" style="visibility: visible">
+    <div class="container-fluid">
+
+        <!-- Brand and toggle get grouped for better mobile display -->
+        <div class="navbar-header">
```
Comment (nisargjhaveri) [inline app/views/auth/login.php]: Why isn't the common navbar used here?

File: app/views/auth/signup.php [hunk 5]
```diff
@@ -1,11 +1,123 @@
...
+    <link href="<?php echo base_url(); ?>static/css/sidebar.css" rel="stylesheet" media="screen">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.2/jquery.min.js"></script>
+    <script src="<?php echo base_url(); ?>static/js/bootstrap.js"></script>
+
+</head>
+<nav class="navbar navbar-inverse navbar-fixed-top" id="navbar" role="navigation" style="visibility: visible">
+    <div class="container-fluid">
+
+        <!-- Brand and toggle get grouped for better mobile display -->
+        <div class="navbar-header">
```
Comment (nisargjhaveri) [inline app/views/auth/signup.php]: Common navbar?

Comment (imjalpreet) [inline app/views/auth/signup.php]: I thought we don't have to show the modules on the login and signup page, do we have to?

Comment (nisargjhaveri) [inline app/views/auth/signup.php]: I think we should. If someone gets here by mistake, he should have options to easily navigate.

Comment (imjalpreet) [inline app/views/auth/signup.php]: In that case, He can go to home and then continue. If u want I will change it.

Comment (nisargjhaveri) [inline app/views/auth/signup.php]: Please change? This would also help us just maintaining one navbar..

**Comment by imjalpreet** [comment]
Done!

**Comment by nisargjhaveri** [comment]
Looks good.

**Pull request merged**